pkg/skaffold/docker/dependencies.go

```diff
@@ -89,19 +89,25 @@ func GetDependencies(ctx context.Context, workspace string, dockerfilePath strin
 }
 
 // readDockerignore reads patterns to ignore
-func readDockerignore(workspace string) ([]string, error) {
+func readDockerignore(workspace string, dockerfilePath string) ([]string, error) {
```

Comment:
Should it be called `absDockerfilePath` to show that it's not relative to the `workspace`?

Reply:
Yes, makes sense. Done.